datadog_lambda/tracing.py

```diff
@@ -313,19 +314,32 @@ def _extract_context_from_eventbridge_sqs_event(event):
     body = json.loads(body_str)
     detail = body.get("detail")
     dd_context = detail.get("_datadog")
+    if is_step_function_event(dd_context):
+        try:
+            return extract_context_from_step_functions(detail, None)
```

Comment:
If SFN -> EventBridge -> SQS -> Lambda has a different input format, can you add a test case? Thanks!